lib/src/instance/get_instance.dart

```diff
@@ -131,12 +144,15 @@ class GetInstance {
       }
     });
 
+    // FIXME: this will not await sequentially as probably is expected
```

Comment:
hello! sorry i don't understand what you meant by this, could you give a little explanation?

Reply:
oh i got it now, the recommended is `for in` in these cases. But why have you chose not to fix this? it would be one more breaking change in the code?

Reply:
@Nipodemos 
Here's a more visible example of the forEach vs for with await:
https://dartpad.dev/8430ebb2f2cc046a9cf2f274aea06ae4

I choose to not fix it cuz I don't know if this is the expected behavior or not (the sequential stuff)

Reply:
i am honestly confused for two reasons:
1 - in your dartpad, the `for in` was not sequential, but it was waiting for the time
2 - `forEach` was sequential, but didn't wait the time asked
![image](https://user-images.githubusercontent.com/11494727/92294512-108f7980-ef03-11ea-96ad-2472cc7e1d00.png)

i'm lost
was this expected output??

Reply:
this is crazy man, forEach have such an unpredicted behavior

Reply:
Oh, sorry, the sequential part refers to the array sequence, not the numerical one

forEach will take 6 seconds to finish all outputs

for in await will take the sum of all seconds to output

If I change this part of the code to be for in await it will start to take theoretically more time to execute.